pkg/core/native/crypto_blspoints.go

```diff
@@ -0,0 +1,84 @@
+package native
+
+import (
+	"fmt"
+
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+)
+
+// Wrappers around bls12381 types that must be used as stackitem.Interop values and
+// implement stackitem.Equatable interface.
+type (
+	G1Affine struct {
+		*bls12381.G1Affine
+	}
+	G1Jac struct {
+		*bls12381.G1Jac
+	}
+	G2Affine struct {
+		*bls12381.G2Affine
+	}
+	G2Jac struct {
+		*bls12381.G2Jac
+	}
+	GT struct {
+		*bls12381.GT
+	}
+)
+
+var (
+	_ = stackitem.Equatable(&G1Affine{})
+	_ = stackitem.Equatable(&G1Jac{})
+	_ = stackitem.Equatable(&G2Affine{})
+	_ = stackitem.Equatable(&G2Jac{})
+	_ = stackitem.Equatable(&GT{})
+)
+
+// Equals implements stackitem.Equatable interface.
+func (a *G1Affine) Equals(other any) bool {
+	b, ok := other.(*G1Affine)
```

Comment:
If you have to do this then maybe a single wrapper would suffice?

Reply:
Yes, fixed.